HorizonEngine/Platform/OpenGL/GLWindow.cpp

```diff
@@ -27,6 +27,7 @@ namespace Hzn
 	void GLWindow::onUpdate()
 	{
 		glfwPollEvents();
+		glfwSwapInterval(getVsync() ? 1 : 0);
```

Comment:
why do you add this?

Reply:
I've have attached this to enable and disable vertical sync. This allows you to look your max frame-rate at you computers refresh rate if you vsync is enabled. Depending on whether you have enabled or disabled it. Your frame-rate is either capped to the refresh rate or uncapped. If you run the sample layer, you will see I have added an FPS counter that would show the difference.

Reply:
It sounds like users can choose whether or not to turn on vertical sync depending on their device, which would help improve the user experience. Well done